packages/react/src/components/output/Output.tsx

```diff
@@ -126,10 +128,10 @@ export const Output = (props: IOutputProps) => {
       };
     }
   }, [kernel]);
```

Comment:
Same comment about signals

Reply:
Thx for @fcollonval for the review and help an this. Fixed with https://github.com/datalayer/jupyter-ui/pull/282/commits/771d78096376ba5f4d007ccc8e4bed81801236ff